.vscode/settings.json

```diff
@@ -1,9 +1,30 @@
 {
```

Comment:
Can you revert the changes here? They don't seem necessary for this PR.

Reply:
Done